src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
```

Comment:
Avoid variable namings like `obj`, which decreases code readability. Choose a more descriptive name for the variable that holds the style object. For example, `styleObject` or `cssStyles` would convey more information about what the variable holds. [CODE STYLE]